src/layouts/TopNav.vue

```diff
@@ -74,4 +81,40 @@
 .navigation li:hover {
     color: #4CAF50;
 }
+
+.navigation .menu {
+    display: none;
+}
+
+@media only screen and (max-width: 840px) {
```

Comment:
When the client is less than or equal to 840px, these rules will allow instead, displaying the UL as a popup menu instead of a row of links.